.github/workflows/release.yaml

```diff
@@ -123,6 +123,47 @@ jobs:
       - name :  docker images publish
         run: |
           make docker-publish-cli
+      
+      - name: Make Release
+        env:
+          VERSION: ${{ steps.version.outputs.version }}
```

Comment:
I don't see the output version specified anywhere in file, you need this as a step before this is used:

```
      - name: Set version
        id: version
        run: echo ::set-output name=version::${GITHUB_REF#refs/*/}
```

So if version is `refs/tags/v1.4.2` then the `version` outputs variable will `v1.4.2`